### ISSUE TITLE
Kickstart directory empty in scap-security-guide-0.1.57-oval-5.10

### ISSUE THREAD

**Issue by phibid** [ComplianceAsCode/content#7618]
#### Description of problem:

Kickstart directory is empty in `scap-security-guide-0.1.57-oval-5.10`, is it normal ? In version 0.1.56, all the ks files are there.

#### SCAP Security Guide Version:

0.1.57

**Comment by ggbecker** [comment]
I believe this line has to be updated to reflect the fact that we created a new folder called `products` in the root directory to hold all products:

https://github.com/ComplianceAsCode/content/blob/9350cb6d5b7e9c57889597085ed6c40eaba36c4b/cmake/SSGCommon.cmake#L1482

```
COMMAND ${CMAKE_COMMAND} -DSOURCE="${CMAKE_SOURCE_DIR}/products/rhel*/kickstart/*-ks.cfg" -DDEST="zipfile/${ZIPNAME}/kickstart" -P "${CMAKE_SOURCE_DIR}/cmake/CopyFiles.cmake"
```

**Comment by ggbecker** [comment]
Fixed by #7646
